### PR TITLE
pavanh007-initial-commit

### PR DESCRIPTION
**Pull request by pavanh007**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6714"></a>

### CODE REVIEW

File: pavanh007-initial-commit [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "name": "Pavan H",
+    "type": "personal",
+    "displayStatsPublic": true,
+    "bio": "Backend developer" | "Tech enthusiast",
+    "avatar": "https://i.ibb.co/vHWZ959/linkedIn.png"
```
Comment (SaraJaoude) [inline pavanh007-initial-commit:6]: this is no longer necessary:
```suggestion
```

File: pavanh007-initial-commit [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+    "name": "Pavan H",
+    "type": "personal",
+    "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline pavanh007-initial-commit:4]: this is no longer necessary:
```suggestion
```

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Please move your file to the `data` folder, add `.json` at the end and make sure that your file name matches your GitHub username exactly. 

Instead of `pavanh007-initial-commit` it should be `data/pavanh007.json`

I would suggest you look at our QuickStart Guide (especially section 7) in Docs: https://linkfree.io/docs/quickstart to help you update this Pull Request.

**Comment by kumarsonsoff3** [comment]
I'll be closing this PR as it seems to be a duplicate of #6738.

--
Also, please note that to add the required changes to a PR, make sure to commit to the same branch the PR is coming from, to avoid creating multiple similar PRs.

**Pull request closed without merging**